src/renderer/renderer.ts

```diff
@@ -175,6 +175,10 @@ const app = new Vue({
             ipcRenderer.send(IpcChannels.downloadUpdate);
         });
 
+        vueEventDispatcher.$on(VueEventChannels.commandlineClearHistory, () => {
+            ipcRenderer.send(IpcChannels.commandlineClearHistory)
+        })
```

Comment:
The `renderer.ts` shouldn't care about a specific plugins. In my opinion, because this is not a generic solution it does not scale properly.